src/plugins/nestedRows/nestedRows.js

```diff
@@ -28,6 +31,14 @@ export class NestedRows extends BasePlugin {
     return PLUGIN_PRIORITY;
   }
 
+  /**
+   * Error message for the wrong data type error.
+   *
+   * @returns {string}
+   */
+  static WRONG_DATA_TYPE_ERROR = 'The Nested Rows plugin requires an Array of Objects as a dataset to be' +
+    ' provided. The plugin has been disabled.';
```

Comment:
I would define it as constant at top of the file, but it's up to you.

Reply:
I was going to, but I figured it would be useful for testing purposes to use a static instead of copy/pasting the error message, but you're probably right 👍